### ISSUE TITLE
[Android] Custom text box does not display.

### ISSUE THREAD

**Issue by DysnomianC** [unoplatform/uno#1534]
## Current behavior

Custom text box class doesn't display. Width, height, visible and enabled are all as expected but nothing visible in ui and nothing to tap on or interact with. Occurs with both emulator and on physical phone.

## Expected behavior

The custom text box is visible and able to be interacted with.

## How to reproduce it (as minimally and precisely as possible)

Create new Uno project.
Create a new class in `App1.Shared` called `MyTextBox`:
```csharp
internal partial class MyTextBox : Windows.UI.Xaml.Controls.TextBox { }
```
In MainPage.xml replace the `TextBlock` node with a new `MyTextBox` node:
```xaml
<local:MyTextBox Text="Hello" Height="50"/>
```
Run the uwp app:
The text box displays and can be interacted with.
Run the android app:
No text box appears.
[App1.zip](https://github.com/unoplatform/uno/files/3603387/App1.zip)

## Environment
Nuget Package:
Package Version(s):
Affected platform(s):
- [ ] iOS
- [x] Android
- [ ] WebAssembly
- [ ] Windows
- [ ] Build tasks

Visual Studio
- [ ] 2017 (version: )
- [x] 2019 (version: 16.2.5)
- [ ] for Mac (version: )

Relevant plugins
- [ ] Resharper (version: )

## Anything else we need to know?
Checking the properties on the `MyTextBox`, Visible is Visible, Enabled is true, height and width are both non-zero.
A regular `Windows.UI.Xaml.Controls.TextBox` displays fine.

**Comment by ghuntley** [comment]
Related to an issue that @davidjohnoliver is working on at the moment regarding styles. The workaround for now is to define a style for the inherited class.

**Comment by DysnomianC** [comment]
I haven't been able to get this workaround to work. 

I've tried defining an empty style like 
```xaml
<Style x:Key="MyTextBoxStyle" TargetType="local:MyTextBox" />
```
I've tried copying the base style for a text box
```xaml
<Style x:Key="MyTextBoxStyle" TargetType="TextBoxAdv">
	<Setter Property="Foreground" Value="{ThemeResource TextControlForeground}"/>
	...
</Style>
```
I'm consuming the style with
```xaml
<local:MyTextBox Text="Hello" Height="50" Style="{StaticResource TextBoxAdvStyle}"/>
```
(The styles are defined in App.xaml) but the text box still doesn't appear on android.

Could you provide and example of how the workaround is supposed to work?

**Comment by jeromelaban** [comment]
This has most likely been fixed by #1766
